test/model/test_models_synthetic.py

```diff
@@ -32,13 +33,18 @@
 batch_size = 32
 context_length = 2
 epochs = 1
+ctx = (
```

Comment:
Couldn't we use `normalize_ctx` here?

Reply:
Can make it to `ctx=normalize_ctx(None)` instead but it's less clear I guess, so being explicit here.